### PR TITLE
i-bem__dom: add shortcut for `this.elem(elemName, boolModName, true)` as `this.elem(elemName, boolModName)` (close #892)

### LINKED ISSUE #892: i-bem__dom: add shortcut for `this.elem(elemName, boolModName, true)` as `this.elem(elemName, boolModName)`

### PR DESCRIPTION
**Pull request by dfilatov**
cc @narqo @veged

### CODE REVIEW

File: common.blocks/i-bem/__dom/i-bem__dom.spec.js [hunk 1]
```diff
@@ -165,6 +165,61 @@ describe('i-bem__dom', function() {
...
+                    { elem : 'e1', mods : { m1 : 'v2' } },
+                    { elem : 'e1', mods : { m1 : 'v1' } },
+                    { elem : 'e1', mods : { m1 : true } },
+                    { elem : 'e2' }
+                ]
+            })));
+            instance = rootNode.bem('block');
+            e1Elems = instance.domElem.find('.block__e1');
+        });
+        afterEach(function() {
```
Comment (narqo) [inline common.blocks/i-bem/__dom/i-bem__dom.spec.js:198]: No empty line before `afterEach()`;

File: common.blocks/i-bem/__dom/i-bem__dom.js [hunk 2]
```diff
@@ -764,7 +764,7 @@ DOM = BEM.decl('i-bem__dom',/** @lends BEMDOM.prototype */{
      * @private
      * @param {String} name Nested element name
      * @param {String} [modName] Modifier name
-     * @param {String} [modVal] Modifier value
+     * @param {String|Boolean} [modVal=true] Modifier value
```
Comment (narqo) [inline common.blocks/i-bem/__dom/i-bem__dom.js]: Are you sure that this change should be here? What about `elem()` method's JSDoc at [#L786](https://github.com/bem/bem-core/pull/903/files#diff-fe894527c21cbc977d795eac412ce183R786)?

**Comment by narqo** [comment]
Looks OK!

File: common.blocks/i-bem/__dom/i-bem__dom.spec.js (hunk 1 above)
Comment (dfilatov) [inline common.blocks/i-bem/__dom/i-bem__dom.spec.js:198]: fixed

File: common.blocks/i-bem/__dom/i-bem__dom.js (hunk 2 above)
Comment (dfilatov) [inline common.blocks/i-bem/__dom/i-bem__dom.js]: fixed

**Pull request merged**